client/app/pages/queries/view.js

```diff
@@ -487,6 +487,7 @@ function QueryViewCtrl(
       component: 'permissionsEditor',
       resolve: {
         aclUrl: { url: `api/queries/${$routeParams.queryId}/acl` },
+        query: $scope.query,
```

Comment:
This same dialog is used for dashboards too. It makes more sense to pass in the id of the user who owns the object.

Reply:
Oh it's being used in dashboards, too! @kravets-levko can you help me fix it so it works on dashboards, too?

Reply:
@arikfr (cc @kocsmy) Fixed